.github/workflows/publish-experiment.yml

```diff
@@ -18,17 +18,18 @@ jobs:
       - name: install
         run: npm ci
       - name: build:experiment
-        run: npm run cdn:build:experiment
+        run: BRANCH_NAME=${{ env.BRANCH_NAME }} npm run cdn:build:experiment
```

Comment:
Where you define BRANCH_NAME in env above, I wonder if it would be a good idea to do:
```yaml
BRANCH_NAME: ${GITHUB_REF_NAME//[^[:alnum:]_-]/_}
```
Branch names could have slashes and I'm not sure what else.